geomstats/geometry/discrete_surfaces.py

```diff
@@ -956,13 +1041,12 @@ def _ivp(self, initial_point, initial_tangent_vec, times):
         next_point = initial_point + initial_tangent_vec
         geod = [initial_point, next_point]
```

Comment:
This is a problem on my PR, or maybe Emmanuel's original PR: I am confused why the next point is `initial_point + initial_tangent_vec`: 

I feel like should be:
```
step = initial_tangent_vec / (len(times) - 1)
next_point = initial_point + step
```

Reply:
@luisfpereira nevermind, there is no bug, I had just not seen the line above that already divides by len(times) -1